devices/ADS1X15.js

```diff
@@ -92,10 +93,12 @@ var REG = {
   LOWTHRESH : 2,
   HITHRESH : 3
 };
-function ADS1X15(i2c) {
+function ADS1X15(i2c,opts={part:"ads1015"}) {
```

Comment:
I'm afraid Espruino doesn't support default arguments, so I don't believe this code will even run on Espruino unless the minifier is smart enough to transpile it